templates/pricing/pro.html

```diff
@@ -9,7 +9,7 @@
 
 {% block content %}
 
-<section id="ua-infra" class="p-strip--image is-deep" style="background-image:url('https://assets.ubuntu.com/v1/18717572-05_suru2_light_glow_8K_edited.png'); background-position: bottom right;">
+<section id="ua-infra" class="p-strip--image is-deep" style="background-image:url('https://assets.ubuntu.com/v1/18717572-05_suru2_light_glow_8K_edited.png'); background-position: 70% bottom;">
```

Comment:
We should make this into a standalone class, ex. `p-strip--suru-light-glow` to avoid using inline styles and keep future uses of it consistent.

Reply:
Well, we tend to do this when it is [a custom background image](https://github.com/canonical/ubuntu.com/search?q=style%3D%22background-image) but I guess I can do put it in a class